AppDevKit/CameraKitExampleViewController.h

```diff
@@ -0,0 +1,13 @@
+//
+//  CameraKitExampleViewController.h
+//  AppDevKit
+//
+//  Created by  Chih Feng Sung on 6/29/17.
+//  Copyright © 2017 Yahoo. All rights reserved.
```

Comment:
Please modify the copyright declaration like the format below, thanks.

    //  Licensed under the terms of the BSD License.
    //  Please see the LICENSE file in the project root for terms.